e2e_test/batch/catalog/pg_class.slt.part

```diff
@@ -0,0 +1,9 @@
+query ITIIT
+SELECT * FROM pg_catalog.pg_class ORDER BY oid;
```

Comment:
When running e2e concurrently, this test might failed. Suggest to add a limit here to make the test stable.
```suggestion
SELECT * FROM pg_catalog.pg_class ORDER BY oid limit 6;
```